.github/workflows/main.yml

```diff
@@ -1,7 +1,5 @@
 name: OctoBot-Binary-CI
 on:
-  schedule:
-    - cron: '0 5 * * 1' # every monday at 5am
```

Comment:
why removing the cron job ?

Reply:
Because it cannot be used anymore if we don't push to an external storage.
Otherwise, we can build it for fun.

Reply:
the idea is also to check that builds keep working, I think we should keep it and not upload binaries

Reply:
ok